### ISSUE TITLE
ENH: Allow default clade to be configured for the root

### ISSUE THREAD

**Issue by LauraU123** [nextstrain/augur#1028]
### Context
In RSV I would like the root to belong to a clade so that any descendants that are not part of any other clade get that clade.

I'm not sure this is possible right now - it's relevant because I have a branch with no mutations inferred by ancestral that I would like to give a clade name. This doesn't seem to work right now as augur clades doesn't take reference as an argument?

An alternative could be to pass the reference to augur clades - it's kind of surprising that we don't pass the reference at all.

**Comment by victorlin** [comment]
@corneliusroemer do you have any insight here?

**Comment by corneliusroemer** [comment]
Yeah this is me in stealth suggesting it - it's for me to investigate what's going on here.

**Comment by jameshadfield** [comment]
It's possible  for root nodes to belong to a clade (see #1199 for some more context). If the root node doesn't have mutations explicitly set then you can supply the root (reference?) sequence(s) in a node-data file. As an example,

```json
{
  "nodes": {
    "rootNodeName": {
      "sequence": "AAAAAAAAAA",
      "aa_sequences": {
        "geneName1": "LLLLLLL"
      }
    }
  }
}
```

would let you use a clade definition TSV such as:

```
clade	gene	site	alt
root_clade_name	nuc	5	A
root_clade_name	geneName1	5	L
```

These data can be supplied to `augur clades` in a separate node-data JSON file of their own if that helps. 

Are there other ways which would be simpler for your use case?

**Comment by jameshadfield** [comment]
I guess we could also implement some syntax (in the clades TSV) instructing it to simply label the root with this clade. This somewhat breaks the approach of `augur clades` as we're no longer using sequence data to inform the clade, but perhaps this exception is appropriate.
